opto-electronic-advances.csl

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Opto-Electronic Advances</title>
+    <title-short>Opto-Electron Adv</title-short>
+    <link href="http://www.zotero.org/styles/opto-electronic-advances" rel="self"/>
+    <link href="http://www.oejournal.org/J/OEA/Channel/15" rel="documentation"/>
+    <author>
+      <email>xuhaofan@hust.edu.cn</email>
+      <name>Xuhao Fan</name>
+    </author>
+    <author>
+      <name>Hui Gao</name>
+      <email>snail.hui@163.com</email>
+    </author>
+    <author>
+      <name>Yuncheng Liu</name>
+      <email>yuncheng_liu@outlook.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <issn>2096-4579</issn>
+    <updated>2020-11-28T01:46:07+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="symbol" delimiter-precedes-last="never" et-al-min="5" et-al-use-first="5" initialize-with=". " name-as-sort-order="all"/>
```

Comment:
`et-al-min="5" et-al-use-first="5"` is the problem -- `et-al-use-first` needs to be at least one lower then `et-al-min`

Reply:
(and you may want to delete it entirely -- the 6/1 set in the bibliography is more plausible and there's no reason to have it in two places)

Reply:
okay, I will try again

Reply:
A little difference between paper-conference and others in the suffix ";"and "."